deploy.sh

```diff
@@ -115,10 +115,18 @@ svn cp "trunk" "tags/$VERSION"
 
 # Fix screenshots getting force downloaded when clicking them
 # https://developer.wordpress.org/plugins/wordpress-org/plugin-assets/
-svn propset svn:mime-type image/png assets/*.png || true
-svn propset svn:mime-type image/jpeg assets/*.jpg || true
-svn propset svn:mime-type image/gif assets/*.gif || true
-svn propset svn:mime-type image/svg+xml assets/*.svg || true
+if test -d "$ASSETS_DIR" && test -n "$(find "$ASSETS_DIR" -maxdepth 1 -name "*.png" -print -quit)"; then
```

Comment:
I'm wondering why we need to check the `$ASSETS_DIR` here, while, inside the condition, we update mime type for images in the `assets` folder? Can we check the `assets` folder directly? And because the `assets` folder always exists, we only need to check if it contains images.
```suggestion
if test -n "$(find assets/ -maxdepth 1 -name "*.png" -print -quit)"; then
```

Reply:
It does not always exist and this is the very reason for the creation of the issue and PR in the first place.

People should be able to have CI for repo's where they don't ship images or assets with their plugins.

Reply:
@Lewiscowles1986 I'm talking about the `assets` folder checkout from svn [here](https://github.com/10up/action-wordpress-plugin-deploy/blob/7f925d0eb8db7baebe42b1982f766f5485ea9f9d/deploy.sh#L46-L50). That folder may not contain any file, but it always exists.

Reply:
When I tried that on my computer it threw the following error.

> find: Expected a positive decimal integer argument to -maxdepth, but got ‘-1’

Reply:
@Lewiscowles1986 Which command did you use for testing? Which line throws that error?

What I really mean when I chimp in here is: we're checking if image files exist in one folder, then we set the mime type for images in a different folder => theoretically, it doesn't sound right to me.

I agree that the current implementation works as expected and possibly never has any issue, the asset files if exist may always be copied from `$ASSETS_DIR` to `$SVN_DIR/assets` [using rsync](https://github.com/Lewiscowles1986/action-wordpress-plugin-deploy/blob/patch-2/deploy.sh#L97). But if `rsync` fails to copy files in an edge case, our command will fail. Also, if in the future, we add a feature that may prevent copying the assets, we will need to update this again.

Alternatively, to be extra careful, we can use something like this:
```
if test -d "$SVN_DIR/assets" && test -n "$(find "$SVN_DIR/assets" -maxdepth 1 -name "*.png" -print -quit)"; then
    svn propset svn:mime-type "image/png" "$SVN_DIR/assets/*.png" || true
fi
```

Again, this is subtle and optional. Given that the current implementation is working fine, I leave the final decision to you and @iamdharmesh

Reply:
Makes more sense now. Thank you for taking the time. I could adjust it to point at the svn folder, but in such a scenario would we not also consider if svn might fail to add the files? It's a piece of string I was not going to tug at. @iamdharmesh any chance you might cast a 👍 or 👎 on needing this?

Reply:
> we're checking if image files exist in one folder, then we set the mime type for images in a different folder => theoretically, it doesn't sound right to me.

Thanks for the note @dinhtungdu. I completely agree with this, we should check for images in the same directory on which we are going to run the mime-type command. Also, @Lewiscowles1986 after considering the edge case given by @dinhtungdu, checking for images in the SVN `assets` directory looks more proper solution to me.

Thanks

Reply:
Right updated, squashed, pushed.